### ISSUE TITLE
Please add indicator fast scroll 

### ISSUE THREAD

**Issue by qw123wh** [Anrimian/music-player#1]
Hi i really like your app.
Please can you add this function

Please can you add the fast scroll indicator
Most manufacturers: Xiaomi, Samsung, Huawei, ... have a similar mechanism in their music system apps.
I leave you this link to make you understand what I mean.
https://github.com/reddit/IndicatorFastScroll

**Comment by Anrimian** [comment]
Hello.
(I actualized repository a little bit)

Few months ago I had implemented fast scroller, but in minimalistic version, without text popup. The only screen without fast scroller is play queue.

So I need to make this question more clear: Do you mean text popup implementation or fast scroller on play queue screen?

**Comment by qw123wh** [comment]
I mean fixed scroller with letters.  I like to view the artists page in alphabetical order and with that letter scroller.  Example I want to find an artist with the letter G. By clicking on the letter G of the scroller all the artists with the letter G are displayed. Or if they are in all the songs and I want to find songs that start with the letter G for example. With the traditional scroller I should scroll down to the letter G. Many manufacturers such as Samsung, Xiaomi, Hauwei have a scroller  with letters and I'd like to have it in your app too. 
This library does what I mean, if you can implement it I would be very grateful.

https://github.com/reddit/IndicatorFastScroll

**Comment by Anrimian** [comment]
Okay, I understood.

I'm sure that I won't make this enabled by default, likely it will be possible to enable-disable in settings.

Also it brings one more question that I need to think about: how to react on selected sort order in case when it is not alphabetical?

**Comment by qw123wh** [comment]
Yes an option in the settings would be perfect for everyone.  This scroller should work for both alphabetical orders A-Z;  and Z-A.  If there is no alphabetical order, the scroller should not be displayed.  However, for any implementation problems, ask the developer of the scroller.
